packages/medusa/src/api/admin/users/route.ts

```diff
@@ -90,3 +90,5 @@ export const POST = async (
 
   res.status(200).json({ user, token })
 }
+
+export const AUTHENTICATE = false
```

Comment:
Question: Does that mean that listing all users does not require to be auth?

Reply:
@adrien2p ah nice catch 👍 This is why I don't like this approach too much 😄